firefox-ios/firefox-ios-tests/Tests/ClientTests/NativeErrorPage/NativeErrorPageStateTests.swift

```diff
@@ -0,0 +1,52 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Redux
+import XCTest
+
+@testable import Client
+
+final class NativeErrorPageStateTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+        DependencyHelperMock().bootstrapDependencies()
+    }
+
+    override func tearDown() {
+        DependencyHelperMock().reset()
+        super.tearDown()
+    }
+
+    func testLoadErrorpageData() {
```

Comment:
i'd split this test case in two methods so we can be complaint with given when then rules. we could have one test case for `testInitialState` and `testReduceIntialiazeAction`